### DISCUSSION TITLE
Whisper dies when I run it in parallel

### DISCUSSION THREAD

**Discussion by xymoget** [openai/whisper#1667] [Q&A]
I am trying to run multiply python scripts that use whisper on Linux. When I run it once it's pretty fast and transcribes my .wav file within 15 seconds. But when I run second file it does nothing. It uses resources, but somewhy it doesnt transcribe my file. Weird fact is that as soon I close one script, the second one successfully transcribes the file. What could be the reason and how can I solve this issue? Python 3.11.5 64bit, Ubuntu 22.04 64 bit, i9-12900K

**Comment by llloo** [comment]
hi, try to upgrade the `tbb` library:
`conda install tbb`
